pkgs/development/libraries/libtelnet/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, autoconf, automake, libtool, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "libtelnet-${version}";
+  version = "0.21";
+
+  src = fetchFromGitHub {
+    owner = "seanmiddleditch";
+    repo = "libtelnet";
+    rev = "45f2d5cfcf383312280e61c85b107285fed260cf";
+    sha256 = "1lp6gdbndsp2w8mhy88c2jknxj2klvnggvq04ln7qjg8407ifpda";
+  };
+
+  buildInputs = [ pkgconfig autoreconfHook autoconf automake libtool zlib];
```

Comment:
`pkgconfig` and `autoreconfHook` should be `nativeBuildInputs`. With `autoreconHook` specified, `autoconf`, `automake` and `libtool` are not needed anymore.